### PR TITLE
Add python 2.6 and python 3.X support

### LINKED ISSUE #7: Python 3 support

### PR DESCRIPTION
**Pull request by levi-rs**
closes #7

### CODE REVIEW

**Comment by levi-rs** [comment]
The other PR should be good to merge, but I'm finding some bugs with this one. Specifically, some of the small differences that changed between py2 and py3, like changing `next()` to `__next__()` for iterators.

**Comment by levi-rs** [comment]
I'm about halfway through verifying client methods still work.

**Comment by travispavek** [comment]
How is this going? I am tempted to merge and fix any remaining regressions in a separate PR.

**Comment by levi-rs** [comment]
Go ahead and merge, and I'll make a new PR if I find anything else.

**Comment by levi-rs** [comment]
Okay, we're good to merge if those CI checks pass.

File: test.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+from __future__ import print_function
```
Comment (travispavek) [inline test.py]: I assume this file was grabbed by accident.  Please remove it.

Comment (levi-rs) [inline test.py]: Removed

**Pull request merged**